tests/validation/cattlevalidationtest/core/test_services.py

```diff
@@ -695,6 +696,7 @@ def check_service_activate_stop_instance_scale(super_client, client,
         logger.info("Stopped container - " + container_name)
         service = client.wait_success(service)
 
+    logger.info("service being updated - " + service.name)
```

Comment:
any chance of the service name having duplicates? If so, should probably use uuid instead